### PR TITLE
docs(core): dynamic page add facets example

### PR DESCRIPTION
**Pull request by genereg1**
## Related Issue(s)

closes https://github.com/SAP/fundamental-ngx/issues/4991

## Description

- Added Facets example to Core Dynamic Page

##### During Implementation

1. Visual Testing:

-   [x] visual misalignments/updates
-   [x] check Light/Dark/HCB/HCW themes
-   [x] RTL/LTR - proper rendering and labeling
-   [x] responsiveness(resize)
-   [x] Content Density (Cozy/Compact/(Condensed))
-   [x] States - hover/disabled/focused/active/on click/selected/selected hover/press state
-   [x] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
-   [x] Mouse vs. Keyboard support
-   [x] Text Truncation

2. API and functional correctness

-   [x] check for console logs (warnings, errors)
-   [x] API boundary values
-   [x] different combinations of components - free style
-   [n/a] change the API values during testing

3. Documentation and Example validations

-   [x] missing API documentation or it is not understandable
-   [x] poor examples
-   [x] Stackblitz works for all examples

4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

##### PR Quality

-   [x] the commit message(s) follows the guideline:
        https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
-   [n/a] tests for the changes that have been done
-   [x] all items on the PR Review Checklist are addressed :
        https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
-   [x] Run npm run build-pack-library and test in external application
-   [n/a] update `README.md`
-   [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)

### CODE REVIEW

File: libs/core/src/lib/dynamic-page/dynamic-page-header/header/dynamic-page-header.component.ts [hunk 1]
```diff
@@ -48,6 +52,9 @@ export class DynamicPageHeaderComponent implements OnInit, AfterViewInit, OnDest
     @ContentChild(BreadcrumbComponent)
     _breadcrumbComponent: BreadcrumbComponent;
 
+    @ContentChild(FacetComponent)
+    _facetComponent: FacetComponent;
```
Comment (N1XUS) [inline libs/core/src/lib/dynamic-page/dynamic-page-header/header/dynamic-page-header.component.ts:56]: Missing @hidden typedoc

Comment (N1XUS) [reply libs/core/src/lib/dynamic-page/dynamic-page-header/header/dynamic-page-header.component.ts:56]: I don't see any usages of this content child component

Comment (genereg1) [reply libs/core/src/lib/dynamic-page/dynamic-page-header/header/dynamic-page-header.component.ts]: Removed

File: libs/core/src/lib/dynamic-page/dynamic-page-header/header/dynamic-page-header.component.ts [hunk 2]
```diff
@@ -36,6 +36,9 @@ export const ActionSquashBreakpointPx = 1280;
     }
 })
 export class DynamicPageHeaderComponent implements OnInit, AfterViewInit, OnDestroy {
+    /** @hidden */
+    collapsed = false;
```
Comment (N1XUS) [inline libs/core/src/lib/dynamic-page/dynamic-page-header/header/dynamic-page-header.component.ts:40]: `_` prefix

**Review by N1XUS — APPROVED** [review]
Good job! Just one tiny issue with property naming

**Pull request merged**